# Remove references to Tangut Unicode Block constants
# This fixes building with older system icu libraries

--- a/third_party/WebKit/Source/platform/text/Character.cpp
+++ b/third_party/WebKit/Source/platform/text/Character.cpp
@@ -227,8 +227,6 @@ bool Character::CanTextDecorationSkipInk
     case UBLOCK_HANGUL_JAMO_EXTENDED_A:
     case UBLOCK_HANGUL_JAMO_EXTENDED_B:
     case UBLOCK_LINEAR_B_IDEOGRAMS:
-    case UBLOCK_TANGUT:
-    case UBLOCK_TANGUT_COMPONENTS:
       return false;
     default:
       return true;
